### PR TITLE
CS162 UCB

### PR DESCRIPTION
**Pull request by samarjeet**
UCB website links will become active in a couple of days. Will update links to assignments and notes then.

### CODE REVIEW

**Comment by prakhar1989** [comment]
Thanks Samarjeet! Let me know when this is ready to merge and I'll do the needful!

**Comment by Sushant** [comment]
I was going to make the same suggestion. I would actually highly recommended video lectures by Prof. John Kubiatowicz (https://www.youtube.com/watch?v=XgQo4JkN4Bw).

**Comment by robbyoconnor** [comment]
Perhaps link to [this](http://inst.eecs.berkeley.edu/~cs162/sp14/) instead?

File: README.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ Courses
 ---
 
 ### Systems
+- [CS162](http://cs162.eecs.berkeley.edu/) **Operating Systems and Systems Programming** *UC Berkeley*
+  - Operating Systems course by the Chair of EECS, UC Berkeley [David Culler](http://www.cs.berkeley.edu/~culler/)
+  - [Youtube Playlist](https://www.youtube.com/playlist?list=PL-XXv-cvA_iAARFmCufZ6XeMPPgAzNSNa) Fall 2014 lectures
 - [18-447](http://www.ece.cmu.edu/~ece447/s14/doku.php?id=start) **Introduction to Computer Architecture** *CMU*
   - Very comprehensive material on Computer Architecture - definitely more than just "introduction". Online material is very user-friendly, even the recitation videos available online. This is the Spring'14 version by professor [Onur Mutlu ](http://users.ece.cmu.edu/~omutlu/).
```
Comment (hamm99922) [inline README.md]: View code

**Comment by samarjeet** [comment]
The link is working now. Please accept my earlier pull request.

**Comment by prakhar1989** [comment]
@samarjeet You need to resolve the merge conflict before I can merge. Also, please add links to assignments and emojis (refer exisiting courses). Lastly, @awelch83 spent a lot of effort in ordering courses under each section, so please make sure you add CS162 at the right place i.e. before CS 186.

Thanks!

**Pull request closed without merging**